### PR TITLE
docs: add component change history in about page - update release process

### PR DESCRIPTION
**Pull request by josedialpad**
## Description
Generation of the CHANGELOG.json as part of the release process using the [@dialpad/semantic-release-changelog-json plugin](https://github.com/dialpad/semantic-release-changelog-json).
This plugin will generate a json version of the CHANGELOG.md file and will include in the release commit.

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [ ] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [ ] Ensure all `gulp` scripts successfully compile.
 - [ ] Update, remove, or extend all affected documentation.
 - [ ] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -117,6 +117,7 @@ export default defineUserConfig({
     '@baseComponents': path.resolve(__dirname, './baseComponents'),
     '@views': path.resolve(__dirname, './views'),
     '@mixins': path.resolve(__dirname, './common/mixins/'),
+    '@root': path.resolve(__dirname, '../../'),
     '@': path.resolve(__dirname, '../'),
```
Comment (braddialpad) [inline docs/.vuepress/config.js]: hmm this is a bit confusing since `@` is also supposed to mean root

Comment (josedialpad) [reply docs/.vuepress/config.js]: yeah `@` alias to the doc root. What about changing to `@projectRoot` or any other suggestion?

Comment (braddialpad) [reply docs/.vuepress/config.js]: Yeah projectRoot is fine

Comment (josedialpad) [reply docs/.vuepress/config.js]: Thanks, renamed https://github.com/dialpad/dialtone/pull/794/commits/5e5a966e42b710e0facb189b94f26a1faf148853

**Review by braddialpad — APPROVED** [review]

**Pull request merged**